tutorials/darcy_thermo_mech/doc/content/workshop/numerical/fvm_overview.md

```diff
@@ -0,0 +1,245 @@
+# New Feature: Finite Volume Method (FVM)
+
+!---
+
+## Advantages of Finite Volume Method
+
++(1) Flexible:+ Works with arbitrary cell types (even if they are mixed in the same problem)
+
++(2) Inherently conservative:+ Uses the integral form of the PDEs. This makes it perfect for
+      applications where conservation laws need to be respected (fluid flows).
+
++(3) "Accurate":+ Can be second-order accurate in space.
+
++(4) Rich literature:+ Widely used in Computational Fluid Dynamics (CFD) and has been researched for decades.
+
+!---
+
+## Approximating the Solution
+
+!row!
+!col! width=50%
+
+!style fontsize=80%
+First, domain $\Omega$ is split into $N_c$ cells ($\Omega_{i},~i=1,...,N_c$).
+
+!media darcy_thermo_mech/fvm_cell.png style=width:100%;background:white;
+
+!col-end!
+
+!col! width=50%
+
+In MOOSE, the cell-centered finite volume approximation ([!citet](moukalled2016finite) and [!citet](jasak1996error)) is used:
+
+- Constant over the cells (value at cell centroid): $u \approx u_C$ if $r \in \Omega_C$
+- Constant over the faces (value at face centroid): $u \approx u_f$ if $r \in \partial\Omega_f$
+
+Error analysis is done using Taylor expansions, we prefer second-order discretization/interpolation schemes.
+
+!col-end!
+
+!row-end!
+
+!---
+
+## FVM using an Advection-Diffusion-Reaction Problem
+
++(1)+ Write the strong form of the equation:
+
+!equation
+-\nabla\cdot k\nabla u + \vec{\beta} \cdot \nabla u + cu - f = 0
+
++(2)+ Integrate the equation over the domain $\Omega$:
+
+!equation
+\int_\Omega \left(\nabla\cdot k\nabla u + \vec{\beta} \cdot \nabla u +cu -f \right) dV = 0
```

Comment:
Missing the '-' for diffusion from the eq above

Reply:
Added, thanks!